cypress/fixtures/appSettings.js

```diff
@@ -85,6 +89,7 @@ export const APP_SETTINGS = [
 export const CAPITAL_FRANCE_SETTING = {
   id: 'id4',
   data: {
+    questionId: 'id4',
```

Comment:
To be sure it's not coincidently correct in the test because it matches the id, I would have something different in this field (ie. `question-Id4`).
Same for all below.